### PR TITLE
go-md2man: new package at 2.0.0

### PR DESCRIPTION
**Pull request by mmatongo**
## Description of package

A markdown to manpage generator
## Installed manifest of package

```
/var/db/kiss/installed/go-md2man/version
/var/db/kiss/installed/go-md2man/sources
/var/db/kiss/installed/go-md2man/manifest
/var/db/kiss/installed/go-md2man/depends
/var/db/kiss/installed/go-md2man/checksums
/var/db/kiss/installed/go-md2man/build
/var/db/kiss/installed/go-md2man/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/man/man1/go-md2man.1
/usr/share/man/man1/
/usr/share/man/
/usr/share/
/usr/bin/go-md2man
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

**Comment by dylanaraps** [comment]
There are two packages in this PR (Brillo got pulled in too).

**Comment by mmatongo** [comment]
> There are two packages in this PR (Brillo got pulled in too).

Fixed the issue

File: community/go-md2man/build [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh -e
+
+export GOFLAGS="-buildmode=pie -mod=vendor -trimpath"
+export CGO_LDFLAGS="$LDFLAGS"
```
Comment (dylanaraps) [inline community/go-md2man/build:4]: Add a newline below last `export`.

File: community/go-md2man/build [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+go build -o go-md2man .
```
Comment (dylanaraps) [inline community/go-md2man/build:5]: Add a newline below this line.

File: community/go-md2man/build (hunk 1 above)
Comment (mmatongo) [reply community/go-md2man/build:4]: done

File: community/go-md2man/build (hunk 2 above)
Comment (mmatongo) [reply community/go-md2man/build:5]: done

**Comment by mmatongo** [comment]
Packages that need `go-md2man` can be compiled without it. It's bloat and essentially useless.

**Pull request closed without merging**